docs/node-delete.directive.md

```diff
@@ -0,0 +1,48 @@
+# Node Delete directive
+
+Delete multiple files and folders
+
+<!-- markdown-toc start - Don't edit this section.  npm run toc to generate it-->
+
+<!-- toc -->
+
+- [Basic Usage](#basic-usage)
+  * [Properties](#properties)
+  * [Events](#events)
+- [Details](#details)
+
+<!-- tocstop -->
+
+<!-- markdown-toc end -->
+
+## Basic Usage
+
+```html
+<adf-toolbar>
+    <button md-icon-button
+            (delete)="documentList.reload()"
+            [adf-delete]="documentList.selection">
+    </button>
+</adf-toolbar>
+
+<adf-document-list #documentList ...>
+ ...
+</adf-document-list>
+```
+
+### Properties
+
+| Name              | Type                | Default | Description                 |
+| ----------------- | ------------------- | ------- | --------------------------- |
+| adf-delete        | MinimalNodeEntity[] | []      | Nodes to delete             |
+| permanent         | boolean             | false   | Permanent delete, no undo action |
+
+### Events
+
+| Name                      | Description                                  |
+| ------------------------- | -------------------------------------------- |
+| delete                    | emitted when delete/undo process is done |
```

Comment:
I don't think should be in the same component the concern to delete and undo it looks two separate directive to me

Reply:
You are right, but this could lead to satellite directives which will do nothing on their own Maybe we would need a service or Command Pattern undo/redo last action.

Reply:
I would prefer to separate it also because I immagine people could decide to have two separate buttons for this action

Reply:
A command pattern could make sense I agree